packages/utils/provider-amt/build.yaml

```diff
@@ -0,0 +1,18 @@
+image: busybox
+unpack: true
+copy:
+  - image: "{{ .Values.labels.repository }}:v{{ .Values.version }}"
+    source: /usr/local/lib/librpc.so
+    destination: /usr/local/lib/librpc.so
+  - image: "{{ .Values.labels.repository }}:v{{ .Values.version }}"
+    source: /usr/local/include/librpc.h
+    destination: /usr/local/include/librpc.h
+  - image: "{{ .Values.labels.repository }}:v{{ .Values.version }}"
+    source: /system/providers/agent-provider-amt
+    destination: /usr/local/system/providers/agent-provider-amt
+steps:
+  - chmod +x /usr/local/system/providers/agent-provider-amt
+includes:
+  - /usr/local/lib/librpc.so
```

Comment:
I'm wondering if this is really required? Cant we embed the libs statically?

Reply:
ill try a build with c go off and see if we can skip it.

Reply:
It looks like this library cannot be embedded but I was able to link it to a static file so we don't need to clear the ld cache.